### ISSUE TITLE
[RTL] The back arrow icon in the Tracking Protection popup menu is reversed

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6546]
See screenshot:
https://drive.google.com/file/d/1pknrS8aOROR7mzhAJ1xz1wU9Pb6vMu8o/view

The arrow icon should be facing right.

Via : https://bugzilla.mozilla.org/show_bug.cgi?id=1634938

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2401)

**Comment by junbio** [comment]
Since the tracking protection popup was redesigned, this issue is no longer relevant and should be closed.

**Comment by lmarceau** [comment]
I'll close this then!
